hack/tools/release/notes.go

```diff
@@ -131,6 +131,9 @@ func run() int {
 		if key != unknown {
 			c.body = c.body[len(firstWord):]
 		}
+		if strings.TrimSpace(c.body) == "" {
+			continue
+		}
 		c.body = fmt.Sprintf("    - %s", strings.TrimSpace(c.body))
```

Comment:
```suggestion
		c.body = fmt.Sprintf("- %s", strings.TrimSpace(c.body))
```

The indentation makes markdown put the items in a quote

Reply:
ah of course